debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresDefaultValueConverter.java

```diff
@@ -76,6 +77,11 @@
         try {
             Object rawDefaultValue = mapper.parse(defaultValue);
             Object convertedDefaultValue = convertDefaultValue(rawDefaultValue, column);
+            if (convertedDefaultValue instanceof Struct) {
+                // Workaround for DBZ-3816
```

Comment:
Did you mean to link to the AK upstream issue here instead?

Reply:
@gunnarmorling Updated